ui/src/components/NavDrawer.js

```diff
@@ -21,6 +20,37 @@ const useStyles = makeStyles(_theme => ({
     fontWeight: 'bold',
     textDecoration: 'underline',
   },
+  logo: {
+    margin: '8px 8px 0 8px',
+    display: 'flex',
+    padding: '4px 4px 0 4px',
+    alignItems: 'center',
+    flexDirection: 'row',
+    height: '60px',
+    [_theme.breakpoints.down('xs')]: {
+      display: 'none',
+    },
+  },
+  logoImage: {
+    transform: 'scale(0.85)',
```

Comment:
I see `scale(0.85)` again here. Was this chosen independently, or if we change one do we need to change the other? If they should be the same, how about a named constant?

Reply:
This one is for the logo in the nav drawer on wide viewports, and the other one is the logo at the top of the page on narrow viewports. They don't necessarily have to be the same size, but for now I think it looks best.